.github/workflows/revive-llvm-test.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
+    paths:
+      - 'LLVM.lock'
+      - 'crates/llvm-builder/**'
+      - '.github/workflows/revive-llvm-test.yml'
+
+jobs:
+  test:
+    runs-on: parity-large
```

Comment:
We will release on Windows and MacOS too, so this probably should be a matrix test with those platforms to avoid any problems during releasing.